i18n-helpers/src/gettext.rs

```diff
@@ -278,7 +278,7 @@ mod tests {
         ]);
         assert_eq!(
             translate("A footnote[^note].\n\n[^note]: More details.", &catalog),
-            "A FOOTNOTE[^note].\n\n[^note]: MORE DETAILS."
+            "A FOOTNOTE\\[^note\\].\n\n[^note]: MORE DETAILS."
```

Comment:
I think this looks wrong since it looks like the translated text disables the foot note?

Reply:
Yeah, it does look wrong. I will try to tackle it this week.

Reply:
Sorry for taking so long. Was able to finally get back to this.

The new version of pulldown-cmark introduced a new way to parse footnotes which is incompatible with the old way. It is still possible to enable the old approach by changing options which I did. The tests are resolved now.

I am not exactly sure why the new approach breaks but I believe it is because the footnote is not parsed as footnote in case it is not able to find the footnote definition. When we split up the events into groups the footnote definition becomes part of a separate group and because of that it seems to break.

Reply:
Alright, thanks for digging into this! I guess either way is fine for us — long-term, we should probably adapt to the new way, in case the old way gets removed. We can deal with that if or when it happens.